README.md

```diff
@@ -10,3 +10,79 @@ Includes:
 # External Buildpacks
 * [CloudFoundry Cloud Native Buildpacks](https://hub.docker.com/r/cloudfoundry/cnb)
 * [Heroku Java Cloud Native Buildpack](https://github.com/heroku/java-buildpack)
+
+# Quick Reference
+- [Create a Buildpack Tutorial](https://buildpacks.io/docs/create-buildpack/) &rarr; Tutorial to get you started on your first Cloud Native Buildpack
+- [Buildpacks.io](https://buildpacks.io/) &rarr; Cloud Native Buildpack website
+- [Pack – Buildpack CLI](https://github.com/buildpack/pack) &rarr; CLI used to consume the builder, along with source code, and construct an OCI image
+- [CNB Tutorial](https://github.com/buildpack/pack/blob/master/docs/tutorials/app-journey.md) &rarr; Tutorial to get you started using `pack`, a `builder`, and your application to create a working OCI image 
```

Comment:
The source of truth for this tutorial is now at buildpacks.io, not the repo. We actually de-duped it just this morning ;)